### PR TITLE
Add sample_concentration min/max

### PR DESCRIPTION
**Pull request by islean**
## Description

Resolves step 1 in the proposed plan in issue #2626 and adds two columns in the Application table, intended to store the minimum (maximum) thresholds given in the sample_concentration column.

### Added

- sample_concentration_minimum and sample_concentration_maximum to Application table.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b add-sample-concentration-thresholds -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by islean** [comment]
Applied revision in stage: 

image

**Comment by islean** [comment]
Also reflects in the web interface:

image

**Comment by islean** [comment]
Downgrade worked:

image

File: alembic/versions/392e49db40fc_add_sample_concentration_min_max.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+down_revision = "db61c62d9bc0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        table_name="application",
+        column=sa.Column(
+            "sample_concentration_minimum",
```
Comment (henrikstranneheim) [inline alembic/versions/392e49db40fc_add_sample_concentration_min_max.py:23]: Where do we standardise the unit? IN the web interface and docs?

**Review by henrikstranneheim — APPROVED** [review]
👍

File: alembic/versions/392e49db40fc_add_sample_concentration_min_max.py (hunk 1 above)
Comment (islean) [reply alembic/versions/392e49db40fc_add_sample_concentration_min_max.py:23]: Good point - would it be preferable to have sample_concentration_minimum_ng_ul here? Or maybe one can fix the web interface to add that as a suffix? @henrikstranneheim

Comment (seallard) [reply alembic/versions/392e49db40fc_add_sample_concentration_min_max.py:23]: I think both sounds good, to specify it in the fe and in the naming of the column here

**Review by seallard — APPROVED** [review]
Great!

File: alembic/versions/392e49db40fc_add_sample_concentration_min_max.py (hunk 1 above)
Comment (henrikstranneheim) [reply alembic/versions/392e49db40fc_add_sample_concentration_min_max.py]: 👍

Comment (islean) [reply alembic/versions/392e49db40fc_add_sample_concentration_min_max.py]: Would a good compromise be to append the unit in the FE and keep the column name without unit? Sort of like this but without units in the column name?
image

Comment (islean) [reply alembic/versions/392e49db40fc_add_sample_concentration_min_max.py]: So here the FE appends the units, whereas the DB only holds the number values. Or will some people go directly to statusdb and not use the web interface and wonder what units are used?

Comment (seallard) [reply alembic/versions/392e49db40fc_add_sample_concentration_min_max.py]: Hm, yeah it probably does not make sense to include the units in the column name in the db. Sounds good 👍

**Comment by islean** [comment]
Deployed to Hasta production and cg-prod-services by IO 2023-10-26:

```
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/isak.ohlsson
done.
Log deploy... done.
cg, version 51.9.0
1d [isak.ohlsson@hasta:~] [P_base] 2m16s $ 
```

**Pull request merged**